### PR TITLE
Add Issue MediaType

### PR DESCRIPTION
**Pull request by VincentRbbmnd**

### CODE REVIEW

File: ruby/lib/trailer_vote/media_types/issue.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+      registrations :issue do
+        view 'create', :create_issue
+
+        versions 1
+      end
+
+    end
+  end
+end
```
Comment (SleeplessByte) [inline ruby/lib/trailer_vote/media_types/issue.rb]: EOF newline

File: ruby/test/trailer_vote/fixtures/application/vnd.trailervote.issue.v1.create+json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      "caller": "TMDb::Ingestion::IngestMovieJob",
+      "args": {
+        "movie": {
+          "id": "12345",
+          "title": "The Testing Movie"
+        }
+      }
+    }
+  }
+}
```
Comment (SleeplessByte) [inline ruby/test/trailer_vote/fixtures/application/vnd.trailervote.issue.v1.create+json]: EOF newline

File: ruby/lib/trailer_vote/media_types/issue.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            attribute :rescue_context do
+              attribute :caller, String
+              attribute :args, ::Hash, allow_empty: true do
+                not_strict
+              end
+            end
+          end
+
+          attribute :issue do
+            merge version_1_base
```
Comment (SleeplessByte) [inline ruby/lib/trailer_vote/media_types/issue.rb:27]: So you don't get back any timestamp information?

File: ruby/test/trailer_vote/media_types/issue_test.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+          end
+        end
+      end
+
+      def test_fixtures
+        assert_fixture_passes_validation Issue
+      end
+    end
+  end
+end
```
Comment (SleeplessByte) [inline ruby/test/trailer_vote/media_types/issue_test.rb]: EOF newline

File: ruby/lib/trailer_vote/media_types/issue.rb (hunk 3 above)
Comment (VincentRbbmnd) [reply ruby/lib/trailer_vote/media_types/issue.rb:27]: `created_at` and `updated_at` are handled by Rails now, so `created_at` is the time the Issue is created on our side.

**Review by SleeplessByte — CHANGES_REQUESTED** [review]
You probably want to add a `link` to self, because then you don't need to update this media type to add update and destroy via API capability.

You also need to add the issues link to the configuration media type or you won't be able to "know" what the URL is to create issues.

**Review by SleeplessByte — APPROVED** [review]

**Pull request merged**